devtools/create_cryo/eden_static.h

```diff
@@ -20,12 +20,10 @@
  *
  */
 
-#include "cryo/defs.h"
-#include "cryo/cryolib.h"
+#pragma once
+#include "eden.h"
 
-namespace Cryo {
-
-Follower followerList[] = {
+Follower followerList[15] = {
 //            char,                 X,  sx, sy,  ex,  ey,bank,
 	{ PersonId::pidGregor,          5, 211,  9, 320, 176, 228,   0,  0 },
```

Comment:
You will get an error here on Linux with gcc. This is a common issue I have seen  lot from developers using Visual Studio as it accepts this code, but using EnumName::EnumValue is not valid C++. So you should remove the PersonId:: prefix on all those here.

Note that the same code currently in engines/cryo/staticdata.cpp compiles because the enum definition in engines/cryo/defs.h is enclosed in a namespace of the same name, which is not the case for the one you added to devtools/create_cryo/eden.h. Adding the namespace in eden.h is another way to fix it.

And you will also need to fix the same errors below for the other enums (PersonMask, Object, Areas, AreaFlag).